src/select.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {findIndex} from 'lodash';
 
-let searchString;
+let searchString = "";
```

Comment:
If not initialized as an empty string, below the first search will be `"undefinedU"` or whatever character you press.